### PR TITLE
Add 'silent' to installer ping

### LINKED ISSUE #452: Add 'silent' to installer ping
See Bugzilla [bug 1597256](https://bugzilla.mozilla.org/show_bug.cgi?id=1597256).

PR incoming shortly.

### PR DESCRIPTION
**Pull request by mhowell-mozilla**
Fixes #452

Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/firefox-installer/install/install.1.parquetmr.txt [hunk 1]
```diff
@@ -52,4 +52,5 @@ message install {
   optional binary attribution (UTF8);
   optional binary profile_cleanup_prompt (UTF8);
   optional boolean profile_cleanup_requested;
+  optional boolean silent;
```
Comment (acmiyaguchi) [inline templates/firefox-installer/install/install.1.parquetmr.txt]: The parquet schemas have fallen out of favor, only updates to the JSON Schemas will show up in changes to the BigQuery tables.

Comment (mhowell-mozilla) [reply templates/firefox-installer/install/install.1.parquetmr.txt]: Can the readme be updated to reflect that? As someone who only interacts with these things occasionally, it's very difficult to keep up with everything that's always changing, and a little bit of updated documentation would go a long way.

**Comment by mhowell-mozilla** [comment]
Thanks! Are you who I should be specifically tagging for review on these kinds of things?

> The parquet schemas have fallen out of favor, only updates to the JSON Schemas will show up in changes to the BigQuery tables.

Can the readme be updated to reflect that? As someone who only interacts with this stuff occasionally, it's difficult to keep up with all the changes that are always happening, and a little bit of updated documentation would go a long way.

**Comment by acmiyaguchi** [comment]
In general, you should be able to request review and get a deploy from someone on the Data Platform team. I'm familiar with the repository, for feel free to tag me if there isn't someone else who has more context.

> Can the readme be updated to reflect that? 

Yes, it can. The ingestion pipeline has changed a bit during our GCP migration, leading to a drift in the documentation. The new process of modifying the schema is simpler though, once documented.

**Pull request merged**